### DISCUSSION TITLE
Purchased Wrong License - Refunds? I mistakenly purchased the PrimeENG license when I really needed the UI Kit License

### DISCUSSION THREAD

**Discussion by fidxdevops** [primefaces/community#1371] [PrimeNG] [answered]
Any ideas? Is this possible to cancel the license?

**Comment by fidxdevops** [comment]
![image](https://github.com/primefaces/community/assets/162500912/eca6e18a-10e9-4a0f-9135-8a53301c06db)

**Comment by nckirik** [comment]
i think the proper channel to solve this issue is directly contacting to the company: https://www.primetek.com.tr/#contact

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Could you please send an email to contact@primetek.com.tr ?
